src/serviceWorker.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
Fixing linting issues does not involve ignoring the problem, you actually need to make your code compliant with the code, unless it is expressed otherwise from the docs. Please do tend to that or give a thorough, convincing explanation on why you choose to ignore the rule. Same goes here, as stated in Milestone 1 yo should¿´ve deleted the unused files from your app. If you are not planning on using service workers, then you don't need the file.